components/net_traits/lib.rs

```diff
@@ -488,6 +490,7 @@ impl ResourceFetchTiming {
             ResourceAttribute::RedirectCount(count) => self.redirect_count = count,
             ResourceAttribute::RequestStart => self.request_start = precise_time_ns(),
             ResourceAttribute::ResponseStart => self.response_start = precise_time_ns(),
+            ResourceAttribute::RedirectStart => self.redirect_start = precise_time_ns(),
```

Comment:
The only values for redirectStart in the spec are 0 or setting it to the existing fetchStart value, so let's use the new enum I suggested previously to implement that. Additionally, since 20.2 says "If the value of redirectStart is not set", we should check if redirect_start is 0 before setting it to fetch_start.